packages/mobile/src/identity/contactMapping.ts

```diff
@@ -109,19 +109,15 @@ function* updateRecipientsCache(
 export function* fetchAddressesAndValidateSaga({ e164Number }: FetchAddressesAndValidateAction) {
```

Comment:
Changes in this file are just a bit of restructuring to allow some useful code to be leverage by `identity/commentEncryption.ts` too